src/sagemaker_training/smdataparallel.py

```diff
@@ -296,6 +301,12 @@ def run(self, wait=True, capture_error=False):
         cmd = self._create_command()
         cmd.extend(super(SMDataParallelRunner, self)._create_command())
         logging_config.log_script_invocation(cmd, self._env_vars)
+
+        exception_classes = (
```

Comment:
can we keep the code style consistent? define it to a empty list and add on top of it, similar to mpi.py